### PR TITLE
user profile

### PR DESCRIPTION
**Pull request by PhongTaKiai**
- close: #7 

https://github.com/ailab-dev/app_buy_sell/assets/117698659/0de89c51-ad8e-483d-a204-f8a61adf4977

### CODE REVIEW

**Comment by rick1919** [comment]
@PhongTaKiai @mafreud 
The UI is LGTM!

File: lib/src/features/my_profile/presentation/my_profile_page.dart [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import 'package:flutter/material.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+
+class MyProfilePage extends HookConsumerWidget {
```
Comment (mafreud) [inline lib/src/features/my_profile/presentation/my_profile_page.dart:4]: When you don't use hook or riverpod, please use Stateless Widget. Please check the other code as well.

**Review by mafreud — CHANGES_REQUESTED** [review]

File: lib/src/features/my_profile/presentation/my_profile_page.dart (hunk 1 above)
Comment (PhongTaKiai) [reply lib/src/features/my_profile/presentation/my_profile_page.dart]: @mafreud fixed, thank you

**Review by mafreud — APPROVED** [review]
LGTM 🎉

**Pull request merged**